src/Estimators/EstimatorManagerNew.h

```diff
@@ -65,7 +65,7 @@ class EstimatorManagerNew
    *  \param[in]  twf    The fully constructed TrialWaveFunction.
    */
   EstimatorManagerNew(Communicate* comm,
-                      EstimatorManagerInput&& emi,
+                      EstimatorManagerInput emi,
```

Comment:
This should not be changed. Call it with an rvale. QMCDriverNew makes the input. by either merging the global and local emi to make a new object copying the local or global emi or providing the default emi if non is in the input.

Reply:
I will restore it. Here is the reason.
The constructor needs to own an Input.
If being passed by value, the move happens at the call site.
If being passed by rvalue, there is no move, emi contained inputs will still be disassembled. But the emi object remains in QMCDriverNew. Since it is passed by rvalue, the caller QMCDriverNew should be aware that emi should not be used after the call site.

Overall, I still find it unsafe to move inputs around. For each individual estimator, it tries to move in its input. Consider the input has already been copied at QMCDriverNew. It should just copy. Then EstimatorManagerInput can be more const protected.

Reply:
We should have a call and talk this through, I think you are right that there are just too many moves with the input object.  In this case QMCDriverNew has done the combination and ideally that is just in the function scope so moving it is fine, but the complications mount.